packages/server/src/hono/routes/messages.ts

```
@@ -133,6 +135,28 @@
                             .object({
                                 response: z.string().describe('Agent response text'),
                                 sessionId: z.string().describe('Session ID used for this message'),
+                                tokenUsage: z
+                                    .object({
+                                        inputTokens: z.number().optional(),
```

ESLint

[error] Zod schema field "inputTokens" is missing .describe() call. Add .describe("description here") for OpenAPI documentation.

(dexto-custom/require-zod-describe)